Maple2.Server.Game/Manager/PartyManager.cs

```diff
@@ -173,6 +173,22 @@ public class PartyManager : IDisposable {
         return true;
     }
 
+    public bool CheckDisband(long characterId) {
+        if (Party == null) {
+            return false;
+        }
+
+        // Check if any other player is online
+        if (!Party.Members.Values.Any(partyMember => partyMember.Info.Online && partyMember.CharacterId != characterId)) {
+            world.Party(new PartyRequest {
```

Comment:
why does this only need to be sent if someone is online? I feel like in all cases, the world server needs to know that the party should no longer exist since it's the source of truth.

Reply:
It does the opposite. It's sent only if nobody is online.

Reply:
okay but if nobody is online, the world server still needs to know right? otherwise the people would still be in the party when they logged back on.

Reply:
I'm not sure I understand.
The function checks if nobody is online. If nobody is online, it sends a World request to disband and remove the party from the game.
The party no longer exists and there's no way someone could login with the party anymore.